pkg/repo/mysql/mysql.go

```diff
@@ -0,0 +1,34 @@
+package mysql
+
+import (
+	"context"
+	"database/sql"
+	"fmt"
+
+	_ "github.com/go-sql-driver/mysql"
+)
+
+type Conf struct {
+	Addr     string
+	User     string
+	Password string
+	Database string
+}
+
+func DefaultConf() (*Conf, error) {
+	return new(Conf), nil
+}
```

Comment:
Maybe DefaultConf should return
```go
	return &Conf{
		Addr:     "localhost:3306",
		User:     "root",
		Database: "translate",
	}
```
And in main we could unmarshal viper to Conf struct. need to think about it. 
Maybe some suggestions are available

Reply:
Think SSOT.